CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,73 @@
+# Code of Conduct
+
+In order to foster a welcoming, positive, and rewarding experience, we as
+community members take an oath to uphold this code of conduct.
+
+*Isn't decent behavior is already expected?* We want to bring the best we can
+to each other as peers, and explicitly documenting our expectations lets us set
+the bar high for ourselves.
+
+## Be excellent to each other
+
+"Excellent" means we:
+
+- **are radically inclusive** to existing members and newcomers looking to
+  learn or participate. Some may raise questions that have been addressed
+  before. Kindly point them to existing discussions. Everyone is new to a
+  project once.
```

Comment:
Radically is reminiscent of politics, and since CoCs are often used as political tools, I believe we ought to rephrase this. How about **wholly inclusive** instead?

Also,
 Everyone **was** new to a project once.

Reply:
Fantastic improvements @JoshuaFern ; keep 'em coming!
Fixed in 1a048116